src/CC/ShellCheck/ShellScript.hs

```diff
@@ -20,9 +22,24 @@ import           System.FilePath.Posix
 
 --------------------------------------------------------------------------------
 
+-- | List of shells the engine should be able to handle.
+validShells :: [BS.ByteString]
+validShells = ["sh", "ash", "dash", "bash", "ksh"]
```

Comment:
@acook I extracted the list that ShellCheck has of valid shells. Bit of a shame they don't expose this. Maybe opening an issue on their repo wouldn't be a bad ideal. What do you think?